temporal-sdk/src/main/java/io/temporal/client/WorkflowStub.java

```diff
@@ -149,6 +149,9 @@ <R> WorkflowUpdateHandle<R> getUpdateHandle(
 
   WorkflowExecution start(Object... args);
 
+  <R> WorkflowExecution startWithOperation(
```

Comment:
Can we document this? I know `signalWithStart ` isn't but that is more a deficiency of that API.